test/io/input/strings.csv

```diff
@@ -0,0 +1,2 @@
+"mary had ";"a";" little lamb"
+"hello";" from";" row";" #2"
```

Comment:
I'm just pretty sure that someone is going to open this file in their editor, a newline will be added and commited... but I guess tests will catch that.